packages/nodejs/src/storage/StorageBackendDiskAsync.ts

```diff
@@ -161,8 +164,9 @@ export class StorageBackendDiskAsync extends MaybeAsyncStorage {
         const files = await readdir(this.#path);
 
         for (const key of files) {
-            if (key.startsWith(contextKeyStart) && !key.includes(".", len)) {
-                keys.push(key.substring(len));
+            const decodedKey = decodeURIComponent(key);
```

Comment:
Confirming, this'll encode the vast majority of characters.  You think that's the way to go just to ensure we never have to worry about filesystem oddities?

Reply:
it is in fact just mimic the sync lib e used before https://github.com/lmaccherone/node-localstorage/blob/master/LocalStorage.coffee#L152 ... and yes with this we should be safe